data/src/command.rs

```diff
@@ -104,6 +107,12 @@ pub fn parse(s: &str, buffer: &Buffer) -> Result<Command, Error> {
             Kind::Kick => validated::<2, 1, true>(args, |[channel, user], [comment]| {
                 Command::Kick(channel, user, comment)
             }),
+            Kind::Raw => validated::<1, 1, true>(args, |[cmd], [args]| {
+                let args = args
+                    .map(|args| args.split_whitespace().map(|s| s.to_string()).collect())
+                    .unwrap_or_default();
+                Command::Raw(cmd, args)
+            }),
```

Comment:
You can skip using the validated function entirely and just use the cmd / args we parse initially into it